diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/WhenMustHaveElseWarnTest.kt

```diff
@@ -0,0 +1,45 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import com.pinterest.ktlint.core.LintError
+import generated.WarningNames
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.WhenMustHaveElseRule
+import org.cqfn.diktat.util.lintMethod
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+class WhenMustHaveElseWarnTest {
```

Comment:
What will happen in when expression on sealed or enum type? Compiler allows not using `else` there, and it would be strange if we mark it as an error. I think we should check only `when` statements and not `when` expressions, because the compiler checks expressions. Please add such test and check how your code will work.